packages/app/components/Nav/NotificationSettingsModal.tsx

```diff
@@ -55,28 +55,50 @@ export function NotificationSettingsModal({
     mutate();
   };
 
+  const handleOk = async () => {
+    const value = await form.validateFields();
+    try {
+      await editProfile(value);
+      form.setFieldsValue(profile);
+      onClose();
+    } catch (e) {
+      if (
+        e.response?.status === 400 &&
+        e.response?.data?.message === "phone number invalid"
+      ) {
+        form.setFields([
+          { name: "phoneNumber", errors: ["Invalid phone number"] },
```

Comment:
low key we should make a json that's in `common` with the error messages we pass around

Reply:
https://github.com/sandboxnu/office-hours/issues/373